### PR TITLE
docs: update keycloak + notion docs

### PR DESCRIPTION
**Pull request by kangmingtay**
## What kind of change does this PR introduce?
* Clean up auth guides for keycloak and notion

@MildTomato just to confirm, are we gonna keep the mappings for the following fields in the auth settings page:
1. `EXTERNAL_KEYCLOAK_URL` -> `Keycloak URL`
2. `EXTERNAL_XXX_CLIENT_ID` -> `Client ID` 
3. `EXTERNAL_XXX_SECRET` -> `Client Secret`

### CODE REVIEW

File: web/docs/guides/auth/auth-notion.mdx [hunk 1]
```diff
@@ -14,51 +14,55 @@ To enable Notion Auth for your project, you need to set up a Notion Application
...
-- Click on `Login` at the top right to log in.
+- Go to [developers.notion.com](https://developers.notion.com/).
+- Click "View my integrations" and login.
 
 ![notion.so](/img/guides/auth-notion/notion.png)
 
-- Once logged in, go to [notion.so/my-integrations](https://notion.so/my-integrations).
+- Once logged in, go to [notion.so/my-integrations](https://notion.so/my-integrations) and create a new integration.
+- When creating your integration, ensure that you select "Public integration" under "Integration type" and "Read user information including email addresses" under "Capabilities".
+- You will need to add a redirect uri, see [Add the redirect uri](### Add the redirect uri)
```
Comment (J0) [inline web/docs/guides/auth/auth-notion.mdx:31]: Should there be a link/uri associated with [Add the redirect uri] ?

File: web/docs/guides/auth/auth-notion.mdx [hunk 2]
```diff
@@ -14,51 +14,55 @@ To enable Notion Auth for your project, you need to set up a Notion Application
...
-### Add your Notion credentials into your Supabase Project
+![notion.so](/img/guides/auth-notion/notion-creds.png)
 
 - Go to your [Supabase Project Dashboard](https://app.supabase.com)
 - In the left sidebar, click the `Authentication` icon (near the top)
 - Click `Settings` from the list to go to the `Authentication Settings` page
-- Enter the final (hosted) URL of your app under `Site URL` (this is important)
 - Under `External OAuth Providers` turn `Notion Enabled` to ON
-- Enter your `Internal Integration Token` saved in the previous step
+- Enter the "OAuth cliet ID" and "OAuth client secret" obtained in the `client id` and `client secret` fields.
```
Comment (J0) [inline web/docs/guides/auth/auth-notion.mdx:65]:
```suggestion
- Enter the "OAuth client ID" and "OAuth client secret" obtained in the `client id` and `client secret` fields.
```

**Review by kiwicopple — APPROVED** [review]

File: web/docs/guides/auth/auth-notion.mdx (hunk 1 above)
Comment (kangmingtay) [reply web/docs/guides/auth/auth-notion.mdx:31]: hmm i'm trying to link to a section in the page actually and it seems this is the way to do it in mdx?

Comment (J0) [reply web/docs/guides/auth/auth-notion.mdx:31]: Ah okay, wasn't aware of this.  Let's go with this then!

**Review by J0 — APPROVED** [review]

**Pull request merged**